coral/src/app/layout/SidebarNavigation.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, Flexbox, Icon } from "@aivenio/design-system";
+import database from "@aivenio/design-system/dist/src/icons/database";
+import codeBlock from "@aivenio/design-system/dist/src/icons/codeBlock";
+import layoutGroupBy from "@aivenio/design-system/dist/src/icons/layoutGroupBy";
+import code from "@aivenio/design-system/dist/src/icons/code";
+import people from "@aivenio/design-system/dist/src/icons/people";
+import list from "@aivenio/design-system/dist/src/icons/list";
+import cog from "@aivenio/design-system/dist/src/icons/cog";
+import data from "@aivenio/design-system/dist/src/icons/console";
+import classes from "src/app/layout/SidebarNavigation.module.css";
+
+const originLocationKlawAngular = window.location.origin;
+function createSidebarListItem({
```

Comment:
Lets make this into a function component.

Reply:
In the case, I would define its role (`<li>`) in the parent and the do
```
<li>
  <SidebarItem ...props>
</li>
```

Reply:
The `li` does contain a lot of attributes. 

```
    <Flexbox
      htmlTag={"li"}
      alignItems={"center"}
      marginBottom={"l1"}
      colGap={"l1"}
      className={
        active ? classes.mainNavigationLinkActive : classes.mainNavigationLink
      }
      paddingLeft={"l2"}
    >
 ```
 
 So we would only return the <a><icon>text</a> from a utility component, which does not really separate a lot of code in the end 🤔 
 
We could probably wrap a <div> around the link (I think divs are ok to be children of il) and give all attributes to that, but that's adding an unnecessary div in the markup. Which in the end is probably ok because it does not affect anything really, other then looking meh. 

... I think I just answered my own question 😂  WILL DO! :D